tests/CMakeLists.txt

```diff
@@ -69,15 +68,6 @@ set(dbcsr_unittest_common_SRCS
   dbcsr_test_multiply.F
   )
 
-# For each test, set a variable testname_SRCS defining the sources of that test
-set(dbcsr_unittest1_SRCS dbcsr_unittest1.F)
-set(dbcsr_unittest2_SRCS dbcsr_unittest2.F)
-set(dbcsr_unittest3_SRCS dbcsr_unittest3.F)
-set(dbcsr_tensor_unittest_SRCS dbcsr_tensor_unittest.F)
-set(dbcsr_tas_unittest_SRCS dbcsr_tas_unittest.F)
-set(dbcsr_test_csr_conversions_SRCS dbcsr_test_csr_conversions.F)
-set(dbcsr_tensor_test_cpp_SRCS dbcsr_tensor_test.cpp)
-
```

Comment:
This was done this way intentionally: to be able to reuse the same loop with tests consisting of multiple source files. On the other hand we never made use of it, so, whatever.